dkist/net/tests/test_attr_walker.py

```diff
@@ -53,6 +98,18 @@ def test_walker_single(all_attrs_classes, api_param_names):
     assert not set(api_param_names[all_attrs_classes]).difference(params[0].keys())
 
 
+def test_boundingbox_containing(boundingbox_param):
+    da.BoundingBox(bottom_left= boundingbox_param[0], top_right= boundingbox_param[1],
```

Comment:
Once you have constructed these classes you need to pass them into `walker.create` and check the output of that function yields the correct arguments for the dataset search API.